hudi-utilities/src/main/java/org/apache/hudi/utilities/sources/JsonKafkaSource.java

```diff
@@ -69,7 +69,14 @@ public JsonKafkaSource(TypedProperties properties, JavaSparkContext sparkContext
 
   private JavaRDD<String> toRDD(OffsetRange[] offsetRanges) {
     return KafkaUtils.createRDD(sparkContext, offsetGen.getKafkaParams(), offsetRanges,
-            LocationStrategies.PreferConsistent()).map(x -> (String) x.value());
+            LocationStrategies.PreferConsistent()).filter(x -> {
+              String msgValue = (String) x.value();
+              //Filter null messages from Kafka to prevent Exceptions
+              if (msgValue == null) {
```

Comment:
`return msgValue != null;`?

Reply:
> `return msgValue != null;`?

yes,this looks more concise, I will modify the code